src/components/FamilyManifestModal/FamilyManifestModalData.js

```diff
@@ -0,0 +1,99 @@
+import { get, difference } from 'lodash';
+import { compose, withProps } from 'recompose';
+import { injectState } from 'freactal/lib/inject';
+
+import { withQuery } from '@arranger/components';
+
+export const familyMemberAndParticipantDataQueryBody = ({ sqon }) => ({
+  query: `
+    query dataTypes($sqon: JSON) {
+      file {
+        aggregations(filters: $sqon) {
+          participants__kf_id {
+            buckets {
+              doc_count
+              key
+            }
+          }
+          participants__family__family_members__kf_id {
+            buckets {
+              doc_count
+              key
+            }
+          }
+        }
+      }
+    }
+  `,
+  variables: { sqon },
+});
+
+export const dataTypeDataQueryBody = ({ familyMemberIds }) => ({
+  query: `
+    query dataTypes($sqon: JSON) {
+      file {
+        aggregations(filters: $sqon) {
+          data_type {
+            buckets {
+              doc_count
+              key
+            }
+          }
+        }
+      }
+    }
+  `,
+  variables: {
+    sqon: {
+      op: 'and',
+      content: [
+        {
+          op: 'in',
+          content: {
+            field: 'participants.kf_id',
+            value: familyMemberIds,
+          },
+        },
+      ],
+    },
+  },
+});
+
+export const generateFamilyManifestModalProps = async ({ api, projectId, sqon }) => {
```

Comment:
a couple of things here, I think a slightly different design might make these functions easier to reuse:
1) `fetcher` is already in the codebase, the `default` export of `services/arranger`, it takes an api.
2) convert `familyMemberAndParticipantDataQueryBody` and `dataTypeDataQueryBody ` to do the actual fetching themselves (rename to `familyMemberAndParticipantDataQuery`, etc...). those functions would take an api / whatever else they need to make the request. export these as well
3) create another function `dataTypeAggregationsQuery` that takes `familyMemberIds` as an argument
4) if you still want this `generateFamilyManifestModalProps` helper to keep some other code clean, simply call the three functions above, with the difference logic / whatnot. that way the queries can all be accessed separately, and you still get this helper function that gives you the props you want in one place.